build.sbt

```diff
@@ -11,7 +11,11 @@ lazy val root = project
       "de.heikoseeberger" %% "akka-http-circe" % "1.18.0",
       "io.circe" %% "circe-core" % "0.9.3",
       "io.circe" %% "circe-generic" % "0.9.3",
-      "io.buildo" %% "wiro-http-server" % "0.6.10"
+      "io.buildo" %% "wiro-http-server" % "0.6.10",
+      "com.typesafe.slick" %% "slick" % "3.3.2",
+      "com.h2database" % "h2" % "1.4.187",
+      "org.slf4j" % "slf4j-nop" % "1.6.4",      
+      "io.circe" %% "circe-java8" % "0.11.1",
```

Comment:
Are you using these two deps?

Reply:
slf4j-nop is necessary to run slick, the other is unnecessary